apps/web/pages/api/recorded-daily-video.ts

```diff
@@ -182,7 +181,7 @@ async function handler(req: NextApiRequest, res: NextApiResponse) {
       },
     });
 
-    if (!booking || !(booking.location === DailyLocationType || booking?.location?.trim() === "")) {
```

Comment:
Edge case:- Removed this location check because sometime even when location is google meet, zoom and there is some error creating room then cal video is used as fallback. so if you record a meeting on cal video then this webhook will not execute because it's location is not cal video